src/main/java/org/example/Main.java

```diff
@@ -1,58 +1,63 @@
-package org.example;
 
-import lombok.Getter;
+package org.example;
 
+import java.io.PrintStream;
 import java.util.Base64;
-
 import java.util.Scanner;
 
-import static java.lang.System.*;
-
 public class Main {
-
-    public static final String CORRECT_NUMBER = "5";
     public static final int MAX_TRIES = 5;
 
-    public static Scanner getScanner() {
-        return new Scanner(in);
+    public Main() {
     }
 
+    public static Scanner getScanner() {
+        return new Scanner(System.in);
+    }
 
     public static void main(String[] args) {
+        User.password = "artom123";
+        User.username = "Artom";
+        User2.password2 = "vlad123";
+        User2.username2 = "Vlad";
```

Comment:
Why use static field from User? Read about difference between class and instance of class